sample.ini

```diff
@@ -1,29 +1,37 @@
 [General]
-
-hostname = 0.0.0.0
 
-portNumber = 8000
+hostname = 0.0.0.0
 
-enableTracing = 0
+portNumber = 8000
+
+enableTracing = 0
 
 ; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
-
+
 ; remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
-
-
-; These options are unlikely to need editing by users
 
-maxCacheEntries = 2000000
 
-mruSymbolStateFile = /tmp/snappy-mru-symbols.json
+; These options are unlikely to need editing by users
+
+maxCacheEntries = 2000000
+
+mruSymbolStateFile = /var/snappy-mru-symbols.json
```

Comment:
can you turn this into 2 sample files: sample-windows.ini and sample-linux.ini? then these config files will work out of the box for both server deployment and for Windows developers who are profiling their local builds with local symbols i.e. https://developer.mozilla.org/en-US/docs/Mozilla/Performance/Profiling_with_the_Built-in_Profiler_and_Local_Symbols_on_Windows#Profiling_local_builds_%28without_using_talos%29